operate/webapp/src/main/java/io/camunda/operate/webapp/elasticsearch/reader/ElasticsearchListenerReader.java

```diff
@@ -20,11 +41,58 @@
 @Component
 public class ElasticsearchListenerReader extends AbstractReader implements ListenerReader {
 
-  @Autowired private ProcessInstanceReader processInstanceReader;
+  private static final Logger LOGGER = LoggerFactory.getLogger(ElasticsearchListenerReader.class);
+
+  @Autowired private JobTemplate jobTemplate;
```

Comment:
Can we use Constructor base autowiring instead of annotation? I think we agreed some weeks before. See [here for details](https://www.baeldung.com/constructor-injection-in-spring#implicit-constructor-injection)